src/problems/RadMarshakDust/test_radiation_marshak_dust.cpp

```diff
@@ -65,15 +65,15 @@ template <> struct RadSystem_Traits<MarshakProblem> {
 	static constexpr double radiation_constant = a_rad;
 	static constexpr double Erad_floor = erad_floor;
 	static constexpr int beta_order = 0;
-	static constexpr bool enable_dust_gas_thermal_coupling_model = dust_on;
 	static constexpr double energy_unit = 1.0;
 	static constexpr amrex::GpuArray<double, n_group_ + 1> radBoundaries = radBoundaries_;
 	static constexpr OpacityModel opacity_model = opacity_model_;
 };
 
 template <> struct ISM_Traits<MarshakProblem> {
-	static constexpr double gas_dust_coupling_threshold = 1.0e-5;
+	static constexpr bool enable_dust_gas_thermal_coupling_model = true;
 	static constexpr bool enable_photoelectric_heating = false;
+	static constexpr double gas_dust_coupling_threshold = 1.0e-5;
```

Comment:
Why 1e-5 here when it's 1e-6 everywhere else?

Reply:
This is because 1e-5 is the smallest number at which this test won't fail. I'll add a comment to explain that.